docs/user_documentation/ref-expressions.md

```diff
@@ -47,17 +47,17 @@ $('myIntAttributeName').gt(3).and($('myIntAttributeName').lt(6)).value()
 | and      | Expression | $('female').and($('pregnant')).value()   | female && pregnant     |
 
 ### Unit operations
-| Operator | Parameters | Example                                  | Description                                             |
-|----------|------------|------------------------------------------|---------------------------------------------------------|
-| unit     | Unit       | $('height').unit('cm')                   | Sets the current value unit to cm                       |
-| toUnit   | Unit       | $('height').unit('m').toUnit('cm')       | Converts the current value based on the change in units |
+| Operator | Parameters | Example                                    | Description                                             |
+|----------|------------|--------------------------------------------|---------------------------------------------------------|
+| unit     | Unit       | $('height').unit('cm')                     | Sets the current value unit to cm                       |
+| toUnit   | Unit       | $('height').unit('m').toUnit('cm').value() | Converts the current value based on the change in units |
 
 ### Other
-| Operator | Parameters | Example                                  | Description                                                                                |
-|----------|------------|------------------------------------------|--------------------------------------------------------------------------------------------|
-| age      | -          | $('dateOfBirth').age()                   | Returns the age based on the date of birth and the current year                            |
-| map      | Object     | $('data').map({0:1, 1:2}).value()        | Maps categories to eachother                                                               |
-| group    | Array      | $('age').group([18, 35, 50, 75]).value() | Produces ranges which are left inclusive, (-∞, 18), [18, 35), [35, 50), [50, 75), [75, +∞)|                                                                |
+| Operator | Parameters | Example                                  | Description                                                           |
+|----------|------------|------------------------------------------|-----------------------------------------------------------------------|
+| age      | -          | $('dateOfBirth').age().value()           | Returns the age based on the date of birth and the current year       |
+| map      | Object     | $('data').map({0:1, 1:2}).value()        | Maps categories to each other                                         |
+| group    | Array      | $('age').group([18, 35, 50, 75]).value() | '18-35'                                                               |
```

Comment:
The previous description is more understandable for me.

Reply:
it is also incorrect. shall I merge them?